ni/protobuf/types/parametric_data.proto

```diff
@@ -0,0 +1,62 @@
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+syntax = "proto3";
+
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+package ni.protobuf.types;
+
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+option csharp_namespace = "NationalInstruments.Protobuf.Types";
+option go_package = "types";
+option java_multiple_files = true;
+option java_outer_classname = "ParametricDataProto";
+option java_package = "com.ni.protobuf.types";
+option objc_class_prefix = "NIPT";
+option php_namespace = "NI\\PROTOBUF\\TYPES";
+option ruby_package = "NI::Protobuf::Types";
+
+message ParametricDataValue {
+  oneof value {
+    double double_value = 1;
+    bool bool_value = 2;
+    string string_value = 3;
+    int32 int_value = 4;
```

Comment:
I would recommend changing all of the field names around `int32` values to also use `int32" in the name. This matches [wrappers.proto](https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/wrappers.proto) and provides better extensibility/consistency if we want to support 64 bit integers in the future.